### DISCUSSION TITLE
Is there any plan to provide mutual conversion tools between HOCON and JSON, yaml, etc.?

### DISCUSSION THREAD

**Discussion by sunpauliz** [he3-app/he3-feedback#295] [Feature Requests]
### One sentence introduction the new tool or features

Convert HOCON to JSON, yaml, etc.

## Link or screenshot
Is there any plan to provide mutual conversion tools between HOCON and JSON, yaml, etc.?

### System you are using

Windows

**Comment by lyzhang1999** [comment]
Thanks for the feedback, here is the issues link: https://github.com/he3-app/he3-feedback/issues/298
